packages/synapse-react-client/src/components/BasePortalCard/ChallengePortalCard/ChallengePortalCard.tsx

```diff
@@ -0,0 +1,220 @@
+      {badges.map((badge, index) => (
+        <Chip
+          key={badge}
+          label={<Typography variant="smallText1">{badge}</Typography>}
+          variant="filled"
+          style={{ backgroundColor: badgeColors[index] }}
+          icon={<Close fontSize="small" />}
+          className={styles.ChallengePortalCard__chip}
+        />
+      ))}
+    </Box>
+  )
+}
+
+const ChallengePortalCardRow = ({
+  row,
+  entityId,
+  indices,
+  sql,
+  borderRadius,
+}: ChallengePortalCardRowProps) => {
+  const {
+    TITLE,
+    CHALLENGE_NAME,
+    DESCRIPTION,
+    LEARN_MORE_LINK,
+    BADGES,
+    REGISTRATION_STATUS,
+    BACKGROUND_IMAGE,
+    BACKGROUND_COLOR,
+  } = indices
+
+  const backgroundImageId = row.values[BACKGROUND_IMAGE] || ''
+  const imageUrl = useImageUrl(backgroundImageId, entityId ?? '')
+
+  const value = row.values[BADGES]
+  let badges: string[]
+  try {
+    const parsed = JSON.parse(value ?? '[]')
+    badges = Array.isArray(parsed) ? parsed.map(String) : [String(parsed)]
+  } catch {
+    badges = value ? [String(value)] : []
+  }
+
+  return (
+    <BasePortalCard
+      borderRadius={borderRadius}
+      backgroundImage={imageUrl}
+      backgroundColor={row.values[BACKGROUND_COLOR] ?? ''}
+      className={styles.ChallengePortalCard__backgroundImage}
+      contentBelowCard={badges.length > 0 && <Chips badges={badges} />}
+    >
+      <Stack className={styles.ChallengePortalCard__sectionContainer}>
+        <Stack className={styles.ChallengePortalCard__topSection}>
+          {row.values[REGISTRATION_STATUS] && (
+            <Chip
+              color="secondary"
+              variant="filled"
+              className={styles.ChallengePortalCard__registrationStatus}
+              sx={{ backgroundColor: row.values[BACKGROUND_COLOR] }}
+              label={row.values[REGISTRATION_STATUS]}
+            />
+          )}
+          <Stack className={styles.ChallengePortalCard__titleSection}>
+            <Typography variant="headline1">{row.values[TITLE]} </Typography>
+            <Typography
+              variant="headline1"
+              className={styles.ChallengePortalCard__challengeName}
+            >
+              {row.values[CHALLENGE_NAME]}
+            </Typography>
+          </Stack>
+        </Stack>
+        <Stack className={styles.ChallengePortalCard__bottomSection}>
+          <Typography variant="smallText1">
+            {row.values[DESCRIPTION]}
+          </Typography>
+          {row.values[LEARN_MORE_LINK] && (
+            <Button
+              className={styles.ChallengePortalCard__learnMoreButton}
+              variant="outlined"
+              endIcon={<ArrowForwardIcon />}
+              href={row.values[LEARN_MORE_LINK]}
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              Learn more
+            </Button>
+          )}
+        </Stack>
+      </Stack>
+    </BasePortalCard>
+  )
+}
+
+const ChallengePortalCard = ({
```

Comment:
Little confused by the names.  `ChallengePortalCard` is actually the entire section, with the header and all `ChallengePortalCardRows`.
Looking at this, I'd assume `ChallengePortalCardRow` is a single `ChallengePortalCard`, and `ChallengePortalCard` would be called something else (like 'NewChallengesSection', and would be a portal specific component)